src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_MediaPlayerElement.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Windows.Media.Core;
+using Windows.UI.Xaml.Controls;
+using static Private.Infrastructure.TestServices;
+
+namespace Uno.UI.RuntimeTests.Tests.Windows_UI_Xaml_Controls
+{
+	[TestClass]
+	[RunsOnUIThread]
+	public partial class Given_MediaPlayerElement
+	{
+		private static readonly Uri TestVideoUrl = new Uri("https://test-videos.co.uk/vids/bigbuckbunny/mp4/h264/720/Big_Buck_Bunny_720_10s_5MB.mp4");
+
+		[TestMethod]
+		public async void When_NotAutoPlay_Source()
```

Comment:
We may need to disable the test for WPF targets. We'll probably need to add an internal property for the skia targets to determine if the control has been initialized properly (the extension could be found).

Reply:
and 🚀

Reply:
the extension-less platforms part of the check is basically non-existing
does it look passable to you?